index.html.theme

```diff
@@ -99,19 +157,29 @@
         a:link,
         a:visited {
             cursor: pointer;
-            color: rgb(255, 255, 255);
+            color: var(--link-color);
             text-decoration: none !important;
         }
 
+        .status-item h3, .icons a {
+            color: var(--text-color) !important;
+        }
+
         a:hover {
             color: rgb(86, 86, 86);
         }
-
     </style>
 </head>
 <body>
-    <h1>TinyStatus</h1>
-    <h2>Current Status</h2>
+    <div class="logo"><img src="assets/apple-touch-icon.png" alt="Logo"/><h1>TinyStatus</h1></div>
+    <div class="icons">
+        <a href="https://github.com/harsxv/tinystatus" target="_blank"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-github"><path d="M15 22v-4a4.8 4.8 0 0 0-1-3.5c3 0 6-2 6-5.5.08-1.25-.27-2.48-1-3.5.28-1.15.28-2.35 0-3.5 0 0-1 0-3 1.5-2.64-.5-5.36-.5-8 0C6 2 5 2 5 2c-.3 1.15-.3 2.35 0 3.5A5.403 5.403 0 0 0 4 9c0 3.5 3 5.5 6 5.5-.39.49-.68 1.05-.85 1.65-.17.6-.22 1.23-.15 1.85v4"/><path d="M9 18c-4.51 2-5-2-7-2"/></svg> GitHub</a>
+        <a href="https://buymeacoffee.com/harsxv" target="_blank"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-coffee"><path d="M10 2v2"/><path d="M14 2v2"/><path d="M16 8a1 1 0 0 1 1 1v8a4 4 0 0 1-4 4H7a4 4 0 0 1-4-4V9a1 1 0 0 1 1-1h14a4 4 0 1 1 0 8h-1"/><path d="M6 2v2"/></svg> Buy Me a Coffee</a>
```

Comment:
Hello @AalokeCode, could you kindly remove the line about the Buy Me a Coffee button? Once that's done, we'll be all set to merge. Thanks!

Reply:
I've also updated the label and linked the issue for you :)

Reply:
Alright! I'll remove it.